weave-js/src/components/ErrorBoundary.tsx

```diff
@@ -20,6 +22,7 @@ export class ErrorBoundary extends Component<Props, State> {
 
   public componentDidCatch(error: Error, errorInfo: ErrorInfo) {
     // TODO: Log to error reporting service?
+    datadogRum.addAction('weave_panel_error_boundary', errorToPayload(error));
```

Comment:
Seems like this is what the TODO on previous line is referring to - can remove it?

Reply:
Done